reproducibility_project/src/engines/cassandra/project.py

```diff
@@ -376,6 +376,71 @@ def run_cassandra(job):
         )
 
 
+@Project.operation
+@Project.pre(cassandra_complete)
+@Project.post(lambda job: "mean_energy_box1" in job.document)
+def statistics(job):
+    """Compute statistical quantities for each job."""
+    import ele
+    from unyt import angstrom, g, mole
+
+    proplist = [
+        "energy_total",
+        "volume",
+        "nmols",
+        "pressure",
+        "density",
+        "mass_density",
+        "energy_angle",
+        "energy_dihedral",
+        "energy_intravdw",
+        "energy_intraq",
+        "energy_inter",
+        "energy_intervdw",
+        "energy_lrc",
+        "energy_interq",
+        "energy_recip",
+        "energy_self",
+        "enthalpy",
+    ]
+
+    NA = 6.032e23
```

Comment:
Avogadro's number is implemented incorrectly here.  I recommend removing its use entirely, but this is what it should be if you insist on using Avogadro's number.  When I correct for the wrong value, Cassandra's results are much closer to those of the other engines.
```suggestion
    NA = 6.032e23
```
```suggestion
    NA = 6.022e23
```